cucumber/authenticators_jwt/features/authn_jwt.feature

```diff
@@ -0,0 +1,63 @@
+Feature: JWT Authenticator - Basic sanity
+
+  In this feature we define an JWT authenticator in policy and perform authentication
+  with Conjur. In successful scenarios we will also define a variable and permit the host to
+  execute it.
+
+  Background:
+    Given I am initialize JWKs endpoint "myJWKs.json"
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: token-app-property
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host myapp
+    """
+    And I am the super-user
+    And I successfully set auth-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set auth-jwt token-app-property variable value "user"
+
+  Scenario: A valid JWT token with identity in the token
+
+    Given I have a "variable" resource called "test-variable"
+    And I permit host "myapp" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "user":"host/myapp",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the audit log file
+    When I authenticate via authn-jwt with the JWT token
+    Then host "myapp" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
```

Comment:
Dont you want to validate that it's content is "test-secret"?

Reply:
Done

Reply:
i am ok with not checking it. this test is for the jwt authenticator. verifying that a secret is correct should be done in `secrets.feature`

Reply:
I agree with @benoaviram as part of this test, you already get the secret just verify you get it right.